### PR TITLE
powertoys: Add version 0.11.0

### PR DESCRIPTION
**Pull request by wenmin92**
[PowerToys](https://github.com/microsoft/PowerToys) is a set of utilities for power users to tune and streamline their Windows experience for greater productivity.

This tools is made by Microsoft.

### CODE REVIEW

File: bucket/powertoys.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "shortcuts": [
+        [
+            "PowerToys.exe",
+            "PowerToys"
+        ]
+    ],
+    "uninstaller": {
+        "script": "Remove-Item \"$Env:LOCALAPPDATA\\Microsoft\\PowerToys\" -Force -Recurse -ErrorAction Ignore"
+    },
+    "checkver": {
```
Comment (Ash258) [inline bucket/powertoys.json]:
```suggestion
    "checkver": "github",
```

File: bucket/powertoys.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "hash": "6ef6e1aee258e21f76ee7973330602b73c15c4dac11d0cf5fad721baa169f0b2",
+    "extract_dir": "PowerToys",
+    "shortcuts": [
+        [
+            "PowerToys.exe",
+            "PowerToys"
+        ]
+    ],
+    "uninstaller": {
+        "script": "Remove-Item \"$Env:LOCALAPPDATA\\Microsoft\\PowerToys\" -Force -Recurse -ErrorAction Ignore"
```
Comment (Ash258) [inline bucket/powertoys.json]:
```suggestion
        "script": "Remove-Item \"$env:LOCALAPPDATA\\Microsoft\\PowerToys\" -Force -Recurse -ErrorAction Ignore"
```

**Comment by Ash258** [comment]
I do not think that removing localappdata is needed on uninstallation. This is use case for lukesampson/scoop#2782

**Comment by wenmin92** [comment]
> I do not think that removing localappdata is needed on uninstallation. This is use case for [lukesampson/scoop#2782](https://github.com/lukesampson/scoop/issues/2782)

Yes, you're right. I will delete this remove action. Actually this app will write a Registry key when setting `start at login`.

**Pull request merged**